include/ua_client_highlevel.h

```diff
@@ -587,6 +587,22 @@ UA_Client_Subscriptions_remove(UA_Client *client, UA_UInt32 subscriptionId);
 UA_StatusCode UA_EXPORT
 UA_Client_Subscriptions_manuallySendPublishRequest(UA_Client *client);
 
+typedef void (*UA_MonitoredEventHandlingFunction)(UA_UInt32 monId,
+                                                  size_t nEventFields,
+                                                  UA_Variant *eventFields,
+                                                  void *context);
+
+UA_StatusCode
+UA_Client_Subscriptions_addMonitoredEvent(UA_Client *client, UA_UInt32 subscriptionId,
+                                          UA_NodeId nodeId, UA_UInt32 attributeID,
+                                          UA_SimpleAttributeOperand *selectClause,
+                                          size_t nSelectClauses,
+                                          UA_MonitoredEventHandlingFunction hf,
+                                          void *hfContext, UA_UInt32 *newMonitoredItemId);
+
```

Comment:
Use const parameters wherever possible and where it makes sense (also for the handling function). (See also: http://embeddedgurus.com/stack-overflow/2009/03/efficient-c-tips-8-use-const/)

My approach is that I first declare every parameter as `const` and then remove it, if there's really a need to change the parameter within the method.